app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,14 @@
 <resources>
-    <string name="app_name">B07 Group 1 Project</string>
+    <string name="app_name">B07 Demo Summer 2024</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="delete_item">Delete Item</string>
+    <string name="add_item">Add Item</string>
+    <string name="back">Back</string>
+    <string name="recyclerview">RecyclerView</string>
+    <string name="scroller">Scroller</string>
+    <string name="spinner">Spinner</string>
+    <string name="manage_items">Manage Items</string>
```

Comment:
Some of this is old can you remove the ones you aren't using?

Reply:
Removed

Reply:
it came back